modules/acl/src/main/java/org/dcache/acl/mapper/AclMapper.java

```diff
@@ -25,26 +25,26 @@
  */
 public class AclMapper {
 
-    private static final Logger logger = LoggerFactory.getLogger("logger.org.dcache.authorization." + AclMapper.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger("LOGGER.org.dcache.authorization." + AclMapper.class.getName());
```

Comment:
we want to change only variable name `logger` to `LOGGER`. The reference in configuration, which is used LoggerFactory.getLogger(...) shoudl stay is.